app/components/hearings-list-for-milestones-list.js

```diff
@@ -68,23 +68,18 @@ export default class HearingsListForMilestonesListComponent extends Component {
     const dispositions = milestone.get('project.dispositions');
     const milestoneParticipantReviewLookup = this.get('milestoneParticipantReviewLookup');
 
-    const filteredDispositionsArray = [];
-
     // Iterate through ALL of the current project's dispositions.
-    // Filter by IF a single disposition's dcpRecommendationsubmittedbyname matches the
+    // Filter by IF a single disposition's fullname matches the
     // current milestone's displayName based on the milestoneParticipantReviewLookup.
-    // disposition.dcpRecommendationsubmittedbyname = e.g. 'QNBP'
-    // disposition.dcpRecommendationsubmittedbyname.substring(2,4) = e.g. 'BP'
+    // disposition.fullname = e.g. 'QN BP'
+    // disposition.fullname.substring(2,4) = e.g. 'BP'
     // matching e.g. 'BP' with the milestoneParticipantReviewLookup provides 'Borough President Review'
-    dispositions.forEach(function(disposition) {
-      // make sure that dcpRecommendationsubmittedbyname is not NULL
-      if (disposition.dcpRecommendationsubmittedbyname) {
-        if (milestoneParticipantReviewLookup[milestone.displayName] === disposition.dcpRecommendationsubmittedbyname.substring(2, 4)) {
-          filteredDispositionsArray.push(disposition);
-        }
-      }
+    const currentMilestoneDispositions = dispositions.filter(function(disposition) {
+      // make sure we aren't grabbing any null values; null values break substring
+      const participantType = disposition.fullname != null ? disposition.fullname.substring(3, 5) : '';
+      return milestoneParticipantReviewLookup[milestone.displayName] === participantType;
     });
-    return filteredDispositionsArray;
+    return currentMilestoneDispositions;
```

Comment:
_great_ — you can even go a step further and just return the `dispositions.filter`.

Reply:
will do :)